docs/src/operate/c8y/apama.md

```diff
@@ -82,7 +82,7 @@ Follow the instructions [here](https://github.com/thin-edge/thin-edge.io_example
 You can test if the project was successfully installed by running the following Apama command:
 
 ```sh
-/opt/softwareag/Apama/bin/apama_env engine_inspect -m
+/opt/cumulocity/Apama/bin/apama_env engine_inspect -m
```

Comment:
I didn't install apama and check the location, but I would trust what c8y guide says :D

> To build the image, run the following command from your installation directory (by default, this is `/opt/cumulocity`):